LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -284,7 +284,7 @@ function viewHistory(groupID){
 
 // convert to title case
 function toTitleCase(str) {
-    return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+    return (str == "" || str == null) ? "" : str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
```

Comment:
Does this tie directly to the email template editor line number overlap? This template file shouldn't be related to the email template editor.